bin/circle/merge-base

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require_relative '../../lib/cdo/git_utils'
+
+`git config --global user.email "circleci-github@example.com"`
+`git config --global user.name "CircleCI"`
+
+`git merge origin/#{GitUtils.circle_pr_branch_base_no_origin}`
```

Comment:
TODO make this handle non-PR runs (introduce helper that catches & sets to staging / default?)